src/backends/gr.jl

```diff
@@ -436,7 +438,7 @@ function gr_viewport_from_bbox(sp::Subplot{GRBackend}, bb::BoundingBox, w, h, vi
         viewport[4] = 0.5 * (vp[3] + vp[4] + extent)
     end
     if hascolorbar(sp)
-        viewport[2] -= 0.1
+        viewport[2] -= gr_colorbar_ratio
```

Comment:
I wanted to make it clear that 0.1 here means the space for colorbar so I defined it as a `const`.  `gr_colorbar_ratio` is used in the code I added to `gr_colorbar` as well.  Are there any other places this value is used?